src/platforms/javascript/guides/react/configuration/integrations/react-router.mdx

```diff
@@ -15,6 +15,8 @@ The React Router integration is designed to work with Sentry Performance Monitor
 
 </Alert>
 
+The React router instrumentation uses the React router library to create `pageload/navigation` transactions to ensure you collect meaninful performance data about the health of your page loads and assoicated requests. Make sure you use a `Router` component combined with `createBrowserHistory` (or equivalent).
```

Comment:
The reference to `createBrowserHistory` doesn't make sense for react router v6. We should either remove this sentence or move it back down with some fluff.